projects/nfdi-gdo-app/src/app/species-selector/species-selector.component.ts

```diff
@@ -92,6 +110,276 @@ export class SpeciesSelectorComponent implements OnInit, OnDestroy {
         'Sympetrum vulgatum',
     ];
 
+    readonly environmentLayers: EnvironmentLayer[] = [
+        {
+            id: '36574dc3-560a-4b09-9d22-d5945f2b8093',
+            name: 'NDVI',
+            colorizer: {
+                type: 'palette',
+                colors: {
+                    '0': [236, 224, 215, 0],
+                    '1': [235, 223, 214, 255],
+                    '2': [234, 222, 212, 255],
+                    '3': [234, 221, 211, 255],
+                    '4': [233, 221, 209, 255],
```

Comment:
Couldn't you get the colorizer from somewhere else, like in the layer list?

Reply:
f87805ac7b020e5a5a7eacab315e5f73cde83ae8